docqueries/or_tools/knapsack.result

```diff
@@ -0,0 +1,51 @@
+BEGIN;
+BEGIN
+SET client_min_messages TO NOTICE;
+SET
+DROP TABLE IF EXISTS knapsack_query;
+NOTICE:  table "knapsack_query" does not exist, skipping
+DROP TABLE
+CREATE TABLE knapsack_query(
+  weight INTEGER,
+  cost INTEGER);
+CREATE TABLE
+INSERT INTO knapsack_query(weight,  cost)
+VALUES
+(12, 4),
+(2, 2),
+(1, 1),
+(4, 10),
+(1, 2);
+INSERT 0 5
+SELECT *
+FROM knapsack_query;
+ weight | cost 
+--------+------
+     12 |    4
+      2 |    2
+      1 |    1
+      4 |   10
+      1 |    2
+(5 rows)
+
+SELECT *
+FROM vrp_knapsack($$SELECT * FROM knapsack_query$$, 15);
+NOTICE:  Entering Knapsack program
+NOTICE:  Starting Execution of inner query
+INFO:  ('Number of rows processed : ', 5)
+NOTICE:  SQL query returned expected column names
+NOTICE:  SQL query returned expected column types
+NOTICE:  Finished Execution of inner query
+INFO:  ('Total value =', 15)
+INFO:  ('Total weight:', 8)
+INFO:  ('Packed items: ', [1, 2, 3, 4])
+INFO:  ('Packed weights: ', [2, 1, 4, 1])
+INFO:  ('Packed values: ', [2, 1, 10, 2])
+NOTICE:  Exiting program
+ vrp_knapsack 
+--------------
+ Success
```

Comment:
Same as above.

Example:
INPUT: `(id, weight, cost)` in the inner query.
OUTPUT: `(id)` or maybe all three columns. I think just id is sufficient because other column values can be found out by SQL JOIN query.

Reply:
I initially implemented something similar to what you are suggesting. but for the other two functions multiple knapsack and bin_packing, this approach won't make sense because each bin should have its own output table so that the user can differentiate which items are in which box

Reply:
When I tried printing the output for each bin one by one, the outputs got merged into one single result query

Reply:
Also Vicky Mentioned something about not wanting to user-defined types. I made some examples on that inside the sql/or_tools/plpython folder. I'll remove that in the final PR.

Reply:
Is there any way to break the result query?? Same columns but different output tables.
Like 
```
 count
-------
   180
(1 row)
 count
-------
   170
(1 row)
```